### ISSUE TITLE
better data split explanation and execution

### ISSUE THREAD

**Issue by gabriben** [gabriben/multilabel#79]
"The authors mention that they have split the datasets mentioned in Table 2 in train, validation and test partitions. It is unclear what was the ratios of the partitions, and if the partitions were stratified (having the same ratio of labels in all partitions). I will assume that the grid search was based on the validation set and the scores in Table 3 are based on the test set, but how the partitions were done must be mentioned. If feasible, crossvalidatiton or nested crossvalidatiton is a good idea to perform."
